### ISSUE TITLE
included scenarios looks obsolete to current gameplay

### ISSUE THREAD

**Issue by FabioPedretti** [lincity-ng/lincity-ng#187]
Take for example good_times scenario.
Here, some substations are not connected to roads, and some others (the ones on the right) not even to a building, so they cannot produce. Not sure if this is just an old game behavior (where substations worked without being connected to roads or buildings) or (unlikey) fixing them is an expected exercise to do after you load the game.
I just report substations example, IIRC there was something else I noticed but don't remember.

**Comment by dbear496** [comment]
You are correct. The starting scenarios were created for an old version of the game when substations and water wells did not need transport.
